Sources/Context.swift

```diff
@@ -22,7 +22,8 @@ public enum StencilContext {
   /// - Throws: `Parameters.Error`
   public static func enrich(context: [String: Any],
                             parameters: [String],
-                            environment: [String: String] = ProcessInfo().environment) throws -> [String: Any] {
+                            environment: [String: String] =
+                              ProcessInfo.processInfo.environment) throws -> [String: Any] {
```

Comment:
You need to either remove two spaces here or move this code back to the previous line and disable the `line_length` rule just this one to allow the CI to pass.